openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/core/PrepareCore.java

```diff
@@ -365,7 +367,21 @@ void contractNodes() {
                 + ", " + Helper.getMemInfo());
     }
 
-
+    private void setTurnRestrictedLevel(int polledNode) {
+        CHEdgeIterator edge1 = vehicleAllExplorer.setBaseNode(polledNode);//FIXME: iterate only over original edges no need to consider shortcuts?
```

Comment:
Whats with this FIXME?

Reply:
Right, it's more of a todo-type of thing. It's correct as it is now, but because turn relations exist only between original edges it is not necessary to include shortcuts in the iterator. I will look into improving it.

Reply:
This has been addressed in https://github.com/GIScience/openrouteservice/pull/866/commits/08aae57583025779e77f8037332585d60fc94958.

Reply:
Thx